package.json

```diff
@@ -19,7 +19,9 @@
   },
   "devDependencies": {
     "code": "^1.5.0",
+    "eslint": "^1.8.0",
```

Comment:
You can read my silly comments in the upstream bug above, but the TL;DR is that `peerDependencies` are noisy in npm@2 and unsupported in npm@3, so not sure if that is the optimal solution.